MAINTAINERS.md

```diff
@@ -16,7 +16,9 @@ can be found [here](https://github.com/tldr-pages/tldr/settings/collaboration).
 
 ## Current members
 
-These people are members of the tldr-pages organization. An automated list can 
+These people are members of the tldr-pages organization. As well as doing 
+everything that a contributor can do, a member also has access to all the 
```

Comment:
Perhaps "a member also has write access", to make this clearer? Anyone can "access" (i.e. read) the repos :)